core/src/main/java/com/hcl/ecomm/core/services/impl/WishListServiceimpl.java

```diff
@@ -0,0 +1,167 @@
+package com.hcl.ecomm.core.services.impl;
+
+import com.google.gson.*;
+import com.hcl.ecomm.core.config.MagentoServiceConfig;
+import com.hcl.ecomm.core.services.WishListService;
+import com.hcl.ecomm.core.services.LoginService;
+import org.apache.commons.httpclient.HttpStatus;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.json.JSONObject;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+@Component(
+        immediate = true,
+        enabled = true,
+        service = WishListService.class)
+public class WishListServiceimpl implements WishListService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(WishListServiceimpl.class);
+
+    @Reference
+    LoginService loginService;
+
+    @Activate
+    private MagentoServiceConfig config;
+
+    @Override
+    public String getDomainName() {
+        return loginService.getDomainName();
+    }
+
+
+    @Override
+    public String getAddToWishListPath() {
+        return config.customer_addToWishList_string();
+    }
+
+    @Override
+    public String getWishListItemsPath() {
+        return config.customer_getWishList_string();
+    }
+
+    private String responseStream = null;
+    private String schema = "http";
+    @Override
+    public JSONObject addToWishList(String Sku, String custToken) {
+        
+        LOG.debug("addToWishList method start  product={}: " + Sku);
+
+        String addTowishlistPath = "";
+        String authToken = "";
+        JSONObject addToWishlistResponse = new JSONObject();;
+
+        try {
+            if(custToken != null && !custToken.isEmpty()) {
+                authToken = custToken;
+                addTowishlistPath = getAddToWishListPath();
+            }
+            String domainName = getDomainName();
+           addTowishlistPath = addTowishlistPath.replace("sku", Sku);
+            String url = schema + "://" + domainName + addTowishlistPath;
+            LOG.info("addTowishlistPath : " + url);
```

Comment:
Remove Log.info or change it to Log.debug

Reply:
updated to Log.debug